### PR TITLE
docs: document setup with read-only config

### PR DESCRIPTION
**Pull request by nodiscc**
Ref. https://github.com/go-gitea/gitea/issues/5704

I don't know what the process to update doc translations is. I could update the french docs if needed.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: docs/content/doc/installation/from-binary.en-us.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ chmod 770 /etc/gitea
 chmod 750 /etc/gitea
 chmod 644 /etc/gitea/app.ini
 ```
+If you don't want the web installer to be able to write the config file at all, it is also possible to make the config file read-only for the gitea user (owner/group `root:root`, mode `0660`), and set `INSTALL_LOCK = true`. In that case all database configuraton details must be prealably set in the config file, as well as the `SECRET_KEY` and `INTERNAL_TOKEN` values. See the [command line documentation](https://docs.gitea.io/en-us/command-line/) for information on using `gitea generate secret INTERNAL_TOKEN`.
```
Comment (jolheiser) [inline docs/content/doc/installation/from-binary.en-us.md]: `prealably` is not an English word.  
I think the intention is `prior` or `beforehand`, so I'd suggest re-wording to something like `All database configuration details must be preset in the config file...`
or
`All database configuration details must be set in the config file beforehand...`

Thanks for the docs update! 😃

**Review by jolheiser — CHANGES_REQUESTED** [review]

File: docs/content/doc/installation/from-binary.en-us.md [hunk 2]
```diff
@@ -82,6 +82,8 @@ chmod 770 /etc/gitea
...
+
```
Comment (axifive) [inline docs/content/doc/installation/from-binary.en-us.md]: Please remove the extra line

**Comment by sapk** [comment]
@nodiscc translation of docs is done directly by updating the file. The french doc is far outdated and a refresh could really be great. Don't hesitate to ping me in your PR if you do so I would gladly review it.

**Comment by nodiscc** [comment]
@jolheiser @axifive Thank you, fixed following your review.

@sapk I will work on it, sure. 

Wouldn't it make sense for the docs to use a localization system like ~~gettext/transifex~~ (looks like https://github.com/go-gitea/gitea/tree/master/options/locale uses https://crowdin.com/) the one already in use by gitea? Else localized docs will keep lagging behind english ones.

Untranslated/out-of-date paragraphs would simply be displayed in english.

A minimalist alternative is a small banner on all localized pages `This documentation may be out of date, please refer to the [english documentation]($pagename.en-us.md) for latest information.`

File: docs/content/doc/installation/from-binary.en-us.md [hunk 3]
```diff
@@ -82,6 +82,7 @@ chmod 770 /etc/gitea
 chmod 750 /etc/gitea
 chmod 644 /etc/gitea/app.ini
 ```
+If you don't want the web installer to be able to write the config file at all, it is also possible to make the config file read-only for the gitea user (owner/group `root:root`, mode `0660`), and set `INSTALL_LOCK = true`. In that case all database configuration details must be set beforehand in the config file, as well as the `SECRET_KEY` and `INTERNAL_TOKEN` values. See the [command line documentation](https://docs.gitea.io/en-us/command-line/) for information on using `gitea generate secret INTERNAL_TOKEN`.
```
Comment (jolheiser) [inline docs/content/doc/installation/from-binary.en-us.md]: Can you use the Hugo `relref` function here?
https://gohugo.io/content-management/cross-references/

`[command line documentation]({{< relref "doc/usage/command-line.en-us.md" >}})`

**Review by jolheiser — CHANGES_REQUESTED** [review]
For consistency, should use Hugo `relref` function.

**Review by lafriks — APPROVED** [review]

**Comment by jolheiser** [comment]
CI is failing.  
My bad, I gave you a wrong filename. It should be `command-line.md` and not `command-line.en-us.md`

**Comment by techknowlogick** [comment]
@jolheiser updated.

**Comment by techknowlogick** [comment]
Seems build is failing from untouched pages. I wonder if version of hugo in image we use has been updated.

**Comment by sapk** [comment]
@techknowlogick I run the test on master and seems fine as the PR branch failed. It maybe related to the relative path.
I think it may need to use a relative `../usage/command-line.md` or an absolute `/doc/usage/command-line.md` relref but that strange since it seems to work on other parts. I will try to look into it.

File: docs/content/doc/installation/from-binary.en-us.md [hunk 4]
```diff
@@ -82,6 +82,7 @@ chmod 770 /etc/gitea
 chmod 750 /etc/gitea
 chmod 644 /etc/gitea/app.ini
 ```
+If you don't want the web installer to be able to write the config file at all, it is also possible to make the config file read-only for the gitea user (owner/group `root:root`, mode `0660`), and set `INSTALL_LOCK = true`. In that case all database configuration details must be set beforehand in the config file, as well as the `SECRET_KEY` and `INTERNAL_TOKEN` values. See the [command line documentation]({{< relref "doc/usage/command-line.md" >}}) for information on using `gitea generate secret INTERNAL_TOKEN`.
```
Comment (sapk) [inline docs/content/doc/installation/from-binary.en-us.md]: It seems that for relref you need to include the language. So this should be like this.
```suggestion
If you don't want the web installer to be able to write the config file at all, it is also possible to make the config file read-only for the gitea user (owner/group `root:root`, mode `0660`), and set `INSTALL_LOCK = true`. In that case all database configuration details must be set beforehand in the config file, as well as the `SECRET_KEY` and `INTERNAL_TOKEN` values. See the [command line documentation]({{< relref "doc/usage/command-line.en-us.md" >}}) for information on using `gitea generate secret INTERNAL_TOKEN`.
```
And rename the file `doc/usage/command-line.md` to `doc/usage/command-line.en-us.md`

**Review by sapk — CHANGES_REQUESTED** [review]
With those changes it should work.

**Pull request merged**